lib/grammers-tl-types/Cargo.toml

```diff
@@ -23,6 +23,8 @@ grammers-tl-gen = { path = "../grammers-tl-gen", version = "0.7.0" }
 grammers-tl-parser = { path = "../grammers-tl-parser", version = "1.1.2" }
 
 [dev-dependencies]
+grammers-tl-gen = { path = "../grammers-tl-gen", version = "0.7.0" }
```

Comment:
Since new test need to run in compile time, dev-dependencies should add crates used by compile-time.